packages/bot-web-ui/src/components/quick-strategy.jsx

```diff
@@ -60,7 +57,7 @@ const QuickStrategyForm = ({
                     ? '342px'
                     : is_keyboard_active
                     ? `calc(100vh - 170px)`
-                    : `calc(100vh - 80px - 35px - 72px)`; // 100vh - Core header - Vertical tab title - Strategy tabs - Footer
+                    : `calc(100% - 70px)`; // 100vh - Core header - Vertical tab title - Strategy tabs - Footer
```

Comment:
- Could we update the comment behind to reflect the calculation?
- Could we rename `form_margin` to `form_height`?

Reply:
> * Could we update the comment behind to reflect the calculation?
> * Could we rename `form_margin` to `form_height`?

We don't need this comment yet so I remove it. 
form_margin updated to form_height